assets/js/components/optin.js

```diff
@@ -34,8 +34,8 @@ class Optin extends Component {
 		super( props );
 
 		this.state = {
-			scriptOnPage: !! googlesitekit.admin.trackingOptin,
-			optIn: !! googlesitekit.admin.trackingOptin,
+			scriptOnPage: !! global.googlesitekitTrackingEnabled,
+			optIn: !! global.googlesitekitTrackingEnabled,
```

Comment:
@tofumatt Why not use `window.googlesitekitTrackingEnabled` in all these references like we're doing elsewhere?

Reply:
I'm guessing because `window` doesn't exist in Node, so this is necessary for Jest.

Reply:
Actually, no real reason other than it's a bit more idiomatic for Webpack/multi-platform (node + browser, as @aaemnnosttv mentioned Jest is running in Node) code. I can change it to `window` and file an issue to move to `global` (which Webpack should translate to `window` anyway in the build), but it more clearly expresses the intent of what we're doing 🤷‍♂

I'll revert for now but file an issue 😄

Reply:
(Filed #920, just for posterity.)